Update ps2000.py

ivannunyadambiz opened the pull request:
added arguments for voltage and current, removed required=True for the group (this lets you change the current or voltage without any other action), fixed help message for info argument

ivannunyadambiz committed:
Update ps2000.py

added arguments for voltage and current, removed required=True for the group (this lets you change the current or voltage without any other action), fixed help message for info argument

marcj71 commented:
Thanks Ivan! The code in 'if name == main' was really only intended as an example, since the functions are usually called from an testing & logging script.
I suggest to only print the message in set_voltage_and_current() when args.verbose == True, and to call the function as ps.set... . If you change that, I'll gladly accept your improvement :)

ivannunyadambiz committed:
Update ps2000.py

don't take this as my final commit. i'm going to have another go at this tomorrow when i'm in the office. I think I found a possible problem or two elsewhere and would like to test and if required, fix it

ivannunyadambiz reviewed (commented):
had a go at it with the supply, and it seems to be doing what it should
